src/pages/news-announcements.tsx

```diff
@@ -45,6 +48,25 @@ const NewsAnnouncements = ({
           <Announcement announcements={announcements} />
         </section>
       )}
+
+      {articles.length > 0 && (
+        <section className={styles.newsCarouselSection}>
+          <NewsCarousel articles={articles} />
+        </section>
+      )}
+
+      {articles.length > 0 && (
```

Comment:
Question: any reason we couldn't combine these two checks they look the same?

Reply:
That's how I initially had it, but had to separate them because react-slick is using some styles (I think the position absolute) on the dots that was preventing me from getting any padding or margin between the two. I'm sure there is a work around we could do in CSS, but couldn't figure it out so thought this would be a passable solution.

Reply:
Interesting. Might be worth a short comment to note that since it's not obvious from the code that was needed. Or maybe placing a `<span/>` in between them would solve it?

Reply:
Didn't think about using a `<span />`. I'll give that a try and if it doesn't work, I'll add in a comment.